include/forcing/WrappedForcingProvider.hpp

```diff
@@ -45,26 +45,61 @@ namespace forcing {
             provider_to_move.wrapped_provider = nullptr;
         }
 
-        const std::vector<std::string> &get_available_forcing_outputs() override {
-            return wrapped_provider->get_available_forcing_outputs();
+        [[deprecated]]
+        const std::vector<std::string> &get_available_forcing_outputs() {
+            return wrapped_provider->get_avaliable_variable_names();
+        }
+
+        /**
+         * @brief Get the avaliable variable names object
+         * 
+         * @return const std::vector<std::string>& the names of avaliable data variables
+         */
+
+        const std::vector<std::string> &get_avaliable_variable_names() override {
+            return wrapped_provider->get_avaliable_variable_names();
         }
 
         /**
          * Get the inclusive beginning of the period of time over which this instance can provide data for this forcing.
          *
          * @return The inclusive beginning of the period of time over which this instance can provide this data.
          */
-        time_t get_forcing_output_time_begin(const std::string &output_name) override {
-            return wrapped_provider->get_forcing_output_time_begin(output_name);
+        [[deprecated]]
+        time_t get_forcing_output_time_begin(const std::string &output_name) {
+            return wrapped_provider->get_data_start_time();
```

Comment:
Yes, for now. We discussed this and it's easier to add a parameterized version to the API later than to take it out if it isn't needed. At present all the scenarios we're working with either have the same time period (Forcing) or maybe the question asked by this method doesn't have a meaningful answer (BMI models, especially with deferred variables). So even stubbing out a variable name parameterized version didn't make much sense.

In the future we may rather want to parameterize such a method with a selector object instead of a string, because we might want to get the begin/end times for a variable name and feature ID (and ?, and ?...)...another reason to defer building it into the API now when it isn't strictly needed.